modules/android/vsandroid_vcxproj.lua

```diff
@@ -326,10 +391,16 @@
 		end
 	end)
 
+	function android.enableEnhancedInstructionSet(cfg)
+		if cfg.vectorextensions == "NEON" then
+			vc2010.element("EnableNeonCodegen", nil, "true")
+		end
+	end
+
 	premake.override(vc2010, "runtimeTypeInfo", function(oldfn, cfg, condition)
 		if cfg.system == premake.ANDROID then
 			-- Note: Android defaults to 'off'
-			if cfg.rtti then
+			if cfg.rtti and cfg.rtti ~= premake.OFF then
```

Comment:
`rtti`, if not set, defaults to "Default" which will be a `true` here, while the comment says the default is `off`, so `false`.

i don't know which one is correct but either the comment is wrong or you need to add a check for "Default".

also you'll need to change almost every test in the `test_android_project` suite to look for the correct behaviour for both `rtti` and `exceptionhandling`.

Reply:
I believe this bit is from @TurkeyMan's PR.